tools/cleanupTarchives.pl

```diff
@@ -243,21 +322,36 @@ sub getTarchiveBasename {
 
 
 =pod
-Function that will identify the duplicate tarchives present in the filesystem.
-- Inputs: - $tarchive_db      = tarchive file stored in the tarchive table of the database
-          - $tarchivesList_db = hash with the list of tarchives stored in the database (keys of the hsh) and their corresponding md5sum (values of the hash) 
-          - tarFileList       = list of tarchives found in the filesystem that matches the basename of $tarchive_db
-- Outputs: - Undef => if no tarchive matching the file stored in the database could be found in the filesystem.
-           - @duplicateTarFiles and $realTarFileFound => if the file stored in the database matches one file of the tarFileList present in the filesystem. 
+
+=head3 identifyDuplicates($tarchive_db, $tarchivesList_db, $tarFileList)
+
+Function that will identify the duplicate DICOM archives present in the filesystem.
+
+INPUTS:
+  - $tarchive_db     : DICOM archive file stored in the database's C<tarchive> table
+  - $tarchivesList_db: hash with the list of DICOM archives location stored in the
+                       database (keys of the hash) and their corresponding md5sum
+                       (values of the hash)
+  - tarFileList      : list of DICOM archives found in the filesystem that matches
```

Comment:
...in the filesystem that __match__ the basename...

Reply:
done